src/Maybe/find.spec.js

```diff
@@ -67,13 +64,11 @@ test('find works with predicates', t => {
 
   t.equal(find(largeNumber, [ 10, '12', 150, 200, 2000 ]).option(0), 150, 'works with predicates')
   t.ok(isSameType(Maybe, isTrue([])), 'returns a Maybe for array value')
-  t.ok(isSameType(Maybe, isTrue(fromArray([]))), 'returns a Maybe for List value')
```

Comment:
we still need this test, this was being used to test that `find` worked with `List`